### ISSUE TITLE
(auto) update from 0.7.6 to 1.1.3 breaks gale

### ISSUE THREAD

**Issue by sam-k0** [Kesomannen/gale#177]
OS: Linux Mint 21.3 Cinnamon
Using Gale AppImage

When I updated gale through the auto-updater, it said it restarts to install the update.
It did not restart, so I manually closed it, which resulted in it not being able to launch anymore ("Gale" is not responding).
I manually installed the newest version, but it didn't help. 
Only after purging all configs and reinstalling, it started again.

Is there any chance this actually was caused by old configs?

**Comment by Kesomannen** [comment]
There may have been some breaking change along the way, not entirely sure. The log would answer that but I'm assuming you don't have that anymore.

No idea about the icons, random Linux issue I guess

**Comment by sam-k0** [comment]
I dont think its a random linux issue.
I had it working perfectly fine but I think I deleted one directory too many, assuming it would re-create them from scratch upon reinstalling. I was wrong.
Is there any way to get gale to refresh the resources like icon cache? The weird thing is that even gale's icon in the top left is broken.
I'd be happy about any help.

**Comment by sam-k0** [comment]
Okay, so I resolved the issue of broken image resources. It really seems to be a random linux issue as I purged 1.1.3 again and reinstalled 0.7.6, now the icons are perfectly fine again. 
I guess I just shouldn't update if I want icons.
Would be cool to figure out at what point / with which version it broke, but I'll leave that for someone else to figure out.
Takeaway seems to be:
- Icons break sometime beyond > 0.7.6
- Autoupdater does not install correctly (Tested the appimage version)
